pkgs/tools/networking/jwhois/default.nix

```diff
@@ -1,14 +1,20 @@
-{stdenv, fetchurl}:
+{stdenv, lynx, fetchurl}:
 
 stdenv.mkDerivation {
+  inherit lynx;
+
   name = "jwhois-4.0";
-  
+  buildInputs = [ lynx ];
+
   src = fetchurl {
     url = mirror://gnu/jwhois/jwhois-4.0.tar.gz;
     sha256 = "0knn7iaj5v0n6jpmldyv2yk4bcy9dn3kywmv63bwc5drh9kvi6zs";
   };
 
-  postInstall = "ln -s jwhois $out/bin/whois";
+  postInstall = ''
+    ln -s jwhois $out/bin/whois
+    sed -i -e "s|/usr/bin/lynx|$lynx/bin/lynx|g" $out/etc/jwhois.conf
```

Comment:
I don't want to nitpick, but you can use
```
sed -i -e "s|/usr/bin/lynx|${lynx}/bin/lynx|g" $out/etc/jwhois.conf
```
(not the curly braces). That way you do not need `inherit lynx;` and the specification of the build inputs.